app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -49,16 +66,24 @@ define(function (require, exports, module) {
           // default image if displayed
           return new ProfileImage();
         })
+        .fin(function () {
+          if (typeof spinnerEl !== 'undefined') {
```

Comment:
@shane-tomlinson should this `isUndefined`?


Ref: https://github.com/mozilla/fxa-content-server/blob/7d8c2c0661c4492895d39cc10ebcbd602f4bf596/app/scripts/views/mixins/password-strength-mixin.js#L21